### ISSUE TITLE
@vercel/remix npm always behind official remix - How to keep in sync with official packages?

### ISSUE THREAD

**Issue by styxlab** [vercel/remix#123]
Currently, official remix packages are on version 2.10.2 (latest), whereas @vercel/remix is still on 2.10.0 (latest). When updating packages it is a bit annoying when @vercel/remix always falls behind. I can understand, if there is a small time lag, but in this case, 2.10.2 has been published more than a week ago.

I thought, there is some automatic procedure that ensures syncing of versions, is that the case? How can this project ensure version updates are published in a timely manner?

**Comment by TooTallNate** [comment]
There [is a workflow](https://github.com/vercel/remix/blob/main/.github/workflow_scripts/sync-repo.js) which is supposed to keep the repo in sync and trigger a publish of the new version when detected. Seems like something isn't quite working right with that. We'll look into that soon.

For now, @vercel/remix v2.10.2 has been published manually.

**Comment by paul-vd** [comment]
@TooTallNate It seems like it's behind again.

```
└─@vercel/remix 2.10.2
 └── ✕ unmet peer @remix-run/dev@2.10.2: found 2.10.3
  ```

**Comment by TooTallNate** [comment]
I (hopefully) fixed the syncing script (6f56b5bfaa071f40c0b53897dd408ba816b67529), and updated `@vercel/remix` for v2.10.3 (1c444c94a5affacbcf1c308e3b4f826263be9932).

Considering the source of this issue (the syncing script) has been now fixed, I will close this issue. Please create a new one if you are still seeing `@vercel/remix` falling behind in the future.
